{{cookiecutter.project_slug}}/production.yml

```diff
@@ -42,6 +42,7 @@ services:
     ports:
       - "0.0.0.0:80:80"
       - "0.0.0.0:443:443"
+      - "0.0.0.0:5555:5555"
```

Comment:
Should this be wrapped inside a conditional to use Celery?

Reply:
Of course, you are right. I missed a lot of things in this PR. I should commit when I am less tired I guess!